### PR TITLE
fix a few non-obvious mistakes in quic where the durable ID's conflic…

### PR DESCRIPTION
**Pull request by chgray**
…t with macros defined elsewhere - these arent obvious because QUIC isnt using manifested ETW everywhere so it doesnt matter.  flipping event names to be clearer

### CODE REVIEW

File: src/manifest/clog.sidecar [hunk 1]
```diff
@@ -10173,6 +10173,118 @@
         }
       ],
       "macroName": "QuicTraceLogInfo"
+    },
+    "RttUpdatedMsg": {
```
Comment (nibanks) [inline src/manifest/clog.sidecar:10177]: Can you "clean" update to remove the old ones that no longer are used?

File: src/core/connection.c [hunk 2]
```diff
@@ -793,7 +793,7 @@ QuicConnUpdateRtt(
     if (RttUpdated) {
         CXPLAT_DBG_ASSERT(Path->SmoothedRtt != 0);
         QuicTraceLogConnVerbose(
-            RttUpdated,
+            RttUpdatedMsg,
```
Comment (nibanks) [inline src/core/connection.c:796]: Instead of `Msg` what about `Evt`?

File: src/manifest/clog.sidecar (hunk 1 above)
Comment (chgray) [reply src/manifest/clog.sidecar:10177]: we shouldnt, at least not in theory - the idea there is that the list grows

File: src/core/connection.c (hunk 2 above)
Comment (chgray) [reply src/core/connection.c:796]: feel free to send a commit;  I like Msg better :)

**Review by nibanks — APPROVED** [review]
Ok. I can create a PR in the future to clean up / possible rename in the future.

**Pull request merged**